experiment/evaluator.py

```diff
@@ -53,6 +53,8 @@ class Result:
   is_semantic_error: bool = False
   semantic_error: str = ''
   triage: str = ''
+  full_textcov: textcov.Textcov = dataclasses.field(
```

Comment:
nit: can we call this `textcov_diff` to make it clearer the existing coverage has already been subtracted? "full_" implies to me that it's not.